client/app/bundles/course/survey/pages/ResponseShow/__test__/index.test.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount } from 'enzyme';
+import ReactTestUtils from 'react-addons-test-utils';
+import CourseAPI from 'api/course';
+import MockAdapter from 'axios-mock-adapter';
+import * as urlHelpers from 'lib/helpers/url-helpers';
+import storeCreator from '../../../store';
+import ResponseShow from '../index';
+
+const client = CourseAPI.survey.responses.getClient();
+const mock = new MockAdapter(client);
+
+const responseData = {
+  response: {
+    id: 5,
+    sections: [{
+      id: 2,
+      weight: 0,
+      title: 'Only section',
+      description: 'Has one question',
+      answers: [{
+        id: 3,
+        question_id: 4,
+        question: {
+          id: 4,
+          question_type: 'text',
+          description: 'Why?',
+          required: true,
+          weight: 0,
+        },
+        text_response: 'Current answer',
+        options: [],
+      }],
+    }],
+  },
+  survey: {
+    id: 6,
+    title: 'Test Response',
+    description: 'Form working?',
+  },
+};
+
+const surveyId = 1;
+const responseId = responseData.response.id;
+urlHelpers.getSurveyId = jest.fn();
+urlHelpers.getSurveyId.mockReturnValue(surveyId.toString());
```

Comment:
I think it's better to mock `window.location` instead, so that we also know if `getSurveyId` is not working.